src/components/RetryGame/index.tsx

```diff
@@ -0,0 +1,34 @@
+import cn from "classnames";
+import { ReactComponent as ReloadIcon } from "../../assets/reload.svg";
+import Container from "../Container";
+import { Action as GameAction } from "../Game/state";
+import IconButton from "../IconButton";
+import styles from "./index.module.css";
+
+type retryGame = {
```

Comment:
This can be named **RetryGameProps** for the sake of maintaining the naming convention.